scripts/admin/users/utils.py

```diff
@@ -24,57 +16,202 @@
         'role': 'Tech Director'
     }
 """
+import os
+import csv
+from google.cloud import firestore
+from dotenv import load_dotenv
+
+from sendgrid import SendGridAPIClient
+from sendgrid.helpers.mail import Mail
+
+load_dotenv()
+
+FIRESTORE_PRJ = 'devcon-london'
+
+SLACK_FILE = 'slack-devconteam-members.csv'
+SLACK_FIELDS = ['username', 'email', 'status', 'billing-active', 'has-2fa', 'has-sso', 'userid', 'fullname', 'displayname']
+
+GOOGLE_GROUP_FILE = 'devconlondon.csv'
+
+MEMBERS_EXPORT_FILE = 'devcon.members.csv'
+MEMBERS_EXPORT_FIELDS = ['uid', 'name', 'email', 'referrer', 'date', 'linkedin', 'github', 'twitter', 'role']
+
+email_body = """<strong>A friendly poke from Devcon.network</strong>
```

Comment:
can automatically send this email to all users who ARE on slack but NOT on website.

Reply:
nice